nautobot/project-static/materialdesignicons-6.5.95/preview.html

```diff
@@ -680,7 +680,7 @@ <h3>Color</h3>
F0DF2",version:"3.5.94"},{name:"plex",hex:"F06BA",version:"1.7.22"},{name:"pliers",hex:"F19A4",version:"6.5.95"},{name:"plus",hex:"F0415",version:"1.5.54"},{name:"plus-box",hex:"F0416",version:"1.5.54"},{name:"plus-box-multiple",hex:"F0334",version:"1.5.54"},{name:"plus-box-multiple-outline",hex:"F1143",version:"4.4.95"},{name:"plus-box-outline",hex:"F0704",version:"1.8.36"},{name:"plus-circle",hex:"F0417",version:"1.5.54"},{name:"plus-circle-multiple",hex:"F034C",version:"1.5.54"},{name:"plus-circle-multiple-outline",hex:"F0418",version:"1.5.54"},{name:"plus-circle-outline",hex:"F0419",version:"1.5.54"},{name:"plus-minus",hex:"F0992",version:"2.4.85"},{name:"plus-minus-box",hex:"F0993",version:"2.4.85"},{name:"plus-minus-variant",hex:"F14C9",version:"5.3.45"},{name:"plus-network",hex:"F041A",version:"1.5.54"},{name:"plus-network-outline",hex:"F0CBA",version:"3.2.89"},{name:"plus-outline",hex:"F0705",version:"1.8.36"},{name:"plus-thick",hex:"F11EC",version:"4.5.95"},{name:"podcast",hex:"F0994",version:"2.4.85"},{name:"podium",hex:"F0D25",version:"3.3.92"},{name:"podium-bronze",hex:"F0D26",version:"3.3.92"},{name:"podium-gold",hex:"F0D27",version:"3.3.92"},{name:"podium-silver",hex:"F0D28",version:"3.3.92"},{name:"point-of-sale",hex:"F0D92",version:"3.4.93"},{name:"pokeball",hex:"F041D",version:"1.5.54"},{name:"pokemon-go",hex:"F0A09",version:"2.5.94"},{name:"poker-chip",hex:"F0830",version:"2.1.19"},{name:"polaroid",hex:"F041E",version:"1.5.54"},{name:"police-badge",hex:"F1167",version:"4.4.95"},{name:"police-badge-outline",hex:"F1168",version:"4.4.95"},{name:"police-station",hex:"F1839",version:"6.2.95"},{name:"poll",hex:"F041F",version:"1.5.54"},{name:"polo",hex:"F14C3",version:"5.3.45"},{name:"polymer",hex:"F0421",version:"1.5.54"},{name:"pool",hex:"F0606",version:"1.5.54"},{name:"popcorn",hex:"F0422",version:"1.5.54"},{name:"post",hex:"F1008",version:"4.0.96"},{name:"post-outline",hex:"F1009",version:"4.0.96"},{name:"postage-stamp",hex:"F0CBB",version:"3.2.89"},{name:"pot",hex:"F02E5",version:"1.5.54"},{name:"pot-mix",hex:"F065B",version:"1.6.50"},{name:"pot-mix-outline",hex:"F0677",version:"1.7.12"},{name:"pot-outline",hex:"F02FF",version:"1.5.54"},{name:"pot-steam",hex:"F065A",version:"1.6.50"},{name:"pot-steam-outline",hex:"F0326",version:"1.5.54"},{name:"pound",hex:"F0423",version:"1.5.54"},{name:"pound-box",hex:"F0424",version:"1.5.54"},{name:"pound-box-outline",hex:"F117F",version:"4.4.95"},{name:"power",hex:"F0425",version:"1.5.54"},{name:"power-cycle",hex:"F0901",version:"2.3.50"},{name:"power-off",hex:"F0902",version:"2.3.50"},{name:"power-on",hex:"F0903",version:"2.3.50"},{name:"power-plug",hex:"F06A5",version:"1.7.12"},{name:"power-plug-off",hex:"F06A6",version:"1.7.12"},{name:"power-plug-off-outline",hex:"F1424",version:"5.2.45"},{name:"power-plug-outline",hex:"F1425",version:"5.2.45"},{name:"power-settings",hex:"F0426",version:"1.5.54"},{name:"power-sleep",hex:"F0904",version:"2.3.50"},{name:"power-socket",hex:"F0427",version:"1.
```

Comment:
Not clear to me what's been changed here but we should *not* be reformatting these files in the final version of this code.